content/integrations/sso/sso-jumpcloud.md

```diff
@@ -69,7 +69,7 @@ You are now ready to import your JumpCloud certificate and metadata to CloudWisd
 2. Upload the **Metadata.xml** file from JumpCloud.
 ![upload-metadata-xml](/images/_index/upload-metadata-xml.png)
 2. When finished, it should look like this:
-![metricly-sso-complete](/images/sso-azure/metricly-sso-complete.png)
```

Comment:
Can you delete the old file or is it still referenced somewhere?

Reply:
The old one was replaced with an updated screenshot of the same name. For jumpcloud, I decided to just add the same image to this page's own /sso-jumpcloud/ folder. I can delete the jumpcloud duplicate and just reference the azure one if that's better.